fedimint-server/src/config/api.rs

```diff
@@ -803,6 +714,26 @@ mod tests {
             }
         }
 
+        /// Helper for awaiting all servers have the status
+        async fn wait_status(&self, status: ServerStatus) {
+            loop {
+                let params = self.client.get_consensus_config_gen_params().await.unwrap();
+                let mismatched: Vec<_> = params
+                    .peers
+                    .iter()
+                    .filter(|(_, param)| param.status != Some(status.clone()))
+                    .collect();
+                if mismatched.is_empty() {
+                    break;
+                }
+                tracing::info!(
+                    target: fedimint_logging::LOG_TEST,
+                    "Test retrying server status"
+                );
+                sleep(Duration::from_millis(10)).await;
```

Comment:
Any thought to backoff on this retry? I see this logging like crazy with a 10ms sleep and log entry every iteration.

Reply:
Just for testing, needs to pass this before DKG finishes, that's why the retry is so low, but it finishes quickly.